opentracing-api/src/test/java/io/opentracing/io/opentracing/mocktracer/MockTracerTest.java

```diff
@@ -0,0 +1,68 @@
+package io.opentracing.io.opentracing.mocktracer;
+
+import io.opentracing.Span;
+import org.junit.Test;
+
+import java.util.List;
+import java.util.Map;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotEquals;
+
+public class MockTracerTest {
+    @Test
+    public void testRootSpan() {
```

Comment:
these tests illustrate the basics of MockTracer... of course the actual instrumentation would not normally live in the test itself.